README.md

```diff
@@ -1 +1,6 @@
-# chat-app
\ No newline at end of file
+# chat-app
+
+### Clone Repository
```

Comment:
Please mention about varaibles to decalred inside .env file. As this file is not added on git user should know which all variables are used throughout the application